pandas/tests/frame/methods/test_select_dtypes.py

```diff
@@ -322,3 +359,14 @@ def test_select_dtypes_typecodes(self):
         expected = df
         FLOAT_TYPES = list(np.typecodes["AllFloat"])
         tm.assert_frame_equal(df.select_dtypes(FLOAT_TYPES), expected)
+
+    def test_select_dtypes_numeric(self):
+        # GH 35340
```

Comment:
can u add the test from the OP

i think was about int / int32

Reply:
hmm so the OP was angling plotting EAs but it's quite old (April 2019)

With `int`/`int32`/`Int32`/`Int64` etc. these snippets pass on master, and even on 1.0.5:
```
In [21]:         ea = pd.array([1, 2], dtype=np.int32) 
    ...:         df = pd.DataFrame(ea) 
    ...:         assert df.select_dtypes(np.number).shape == df.shape                                                                              

```
but I'm not sure we test this so might add that here

Reply:
the ExtensionDtype derived class doesn't pass on master so I think that's what's being fixed here

Reply:
Added those